src/console/App.js

```diff
@@ -30,16 +35,31 @@ const Homepage = props => (
   </div>
 );
 
-class App extends Component {
-  state = {
-    userInfo: null,
-    accessToken: null,
+@connect(
+  (state, props) => ({
+    userInfo: getUserInfo(state),
+    accessToken: getAccessToken(state),
+  }),
+  {
+    userLogin: userLogin,
+    userLogout: userLogout,
+    setUserInfo: setUserInfo,
```

Comment:
We can rewrite this object as `{ userLogin, userLogout, setUserInfo }` since the key names match the variable names (this is a new javascript feature, you can read more on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions)).